File: factorial.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin>>n;
    int factorial=1;
    for(int i=1;i<=n;i++)
    {
        factorial*=i;

    }
    cout<<"factorial of "<<n<<" 5is :"<<factorial<<endl;
}

File: factorial.py
def factorial(n):
    result = 1
    # for i in range(1,n+1):
    #     result*=i
    if n>0:
        result=n*factorial(n-1)
    
    return result

nums_list = list(map(int,input("enter the numbers to which you need to find the factorial:").strip().split()))
print(nums_list)
for i in nums_list:
    print(f"the factorial of {i} is {factorial(i)}")
    
    